src/SIM/SIMbase.C

```diff
@@ -1582,14 +1582,21 @@ void SIMbase::getBoundaryNodes (int pcode, IntVec& glbNodes, Vec3Vec* XYZ) const
     if (abs(p->pindx) == pcode && (pch = this->getPatch(p->patch)))
       switch (pch->getNoParamDim() - abs(p->ldim)) {
       case 1: // The boundary is of one dimension lower than the patch
-        pch->getBoundaryNodes(abs(p->lindx),glbNodes);
+      {
+        IntVec glbNodes2;
+        pch->getBoundaryNodes(abs(p->lindx),glbNodes2);
+        for (const auto& it : glbNodes2)
+          if (std::find(glbNodes.begin(), glbNodes.end(), it) == glbNodes.end())
+            glbNodes.push_back(it);
+
```

Comment:
nei, det er for sammenhengende render. duplikater må fjernes. har ingenting med basiser og gjøre.